### ISSUE TITLE
Automatically add Original Title as Show Alias

### ISSUE THREAD

**Issue by ItsME6969** [Sonarr/Sonarr#5220]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your feature request related to a problem? Please describe

I've always found it oddly funny that I have to submit an alias request for the `original title` of a show. This of course is the case for non-english shows. Is there a way that you guys can implement a system for foreign shows to `always` at minimum have the original title as an alias?

### Describe the solution you'd like

Original Title should `always` be an Alias and that will save energy and time as alias requests does also take weeks for review and approval.

### Describe alternatives you've considered

.

### Anything else?

![Screenshot 2022-11-03 134027 - Copy](https://user-images.githubusercontent.com/89707006/199807735-a9ceaf33-87eb-4846-a03c-59af794b196c.png)

**Comment by bakerboy448** [comment]
Already covered in other GHI and regardless blocked by TVDb's api

https://github.com/Sonarr/Sonarr/issues/5143
https://github.com/Sonarr/Sonarr/issues/269

**Comment by ItsME6969** [comment]
Is it a skyhook or tvdb block? Just looking at their endpoints it looks like it should support all that.
https://thetvdb.github.io/v4-api/#:~:text=the%20primary%20search%20string%2C%20which%20can%20include%20the%20main%20title%20for%20a%20record%20including%20all%20translations%20and%20aliases.

**Comment by ItsME6969** [comment]
Gotcha. But please leave this open as I feel this is a different request by itself. In the other feature request I was requesting to have all that metadata data locally to be enhance filtering while in this one I'm requesting for Aliases to be added automatically to save everyone some time.
